lab2.js

```diff
@@ -110,28 +175,117 @@ assert((new Human()).sayHello(new Klingon()) === "nuqneH",
 // assertions for each one
 //*********************************************************
 function lastLetterSort(stringArray) {
-  function byLastLetter(array) {
+  function byLastLetter(a, b) {
     //TODO: implement me. sort the strings in alphabetical
     // order using their last letter
     // read this: http://www.w3schools.com/jsref/jsref_sort.asp
+    return a.slice(-1).toLowerCase().localeCompare(b.slice(-1).toLowerCase());
```

Comment:
localeCompare works here; it's also worth checking out how to do a compare function you could call with .sort().